utils/metrics/metrics/src/main/java/com/ea/orbit/metrics/MetricsManager.java

```diff
@@ -100,9 +103,9 @@ public synchronized void initializeMetrics(List<ReporterConfig> reporterConfigs)
         }
         else
         {
-            if (logger.isDebugEnabled())
+            if (logger.isWarnEnabled())
             {
-                logger.debug("Attempting to initialize the Metrics Manager when it is already initialized!");
+                logger.warn("Attempting to initialize the Metrics Manager when it is already initialized!");
```

Comment:
@JoeHegarty I've put this back to a WARN as it now is a suspicious state that should be flagged. This change prevents the spam seen in tests anyway :)